features/admin.roles.v2/components/role-list.tsx

```diff
@@ -292,14 +293,19 @@ export const RoleList: React.FunctionComponent<RoleListProps> = (props: RoleList
                 renderer: "semantic-icon"
             },
             {
-                hidden: (role: RolesInterface) => isSubOrg ||
-                    (role?.displayName === CommonRoleConstants.ADMIN_ROLE ||
+                hidden: (role: RolesInterface) => {
+                    return isSubOrg
+                    || role?.meta?.systemRole
+                    || (
+                        role?.displayName === CommonRoleConstants.ADMIN_ROLE ||
                         role?.displayName === CommonRoleConstants.ADMIN_GROUP ||
-                        role?.displayName === administratorRoleDisplayName)
+                        role?.displayName === administratorRoleDisplayName
+                    )
                     || !isFeatureEnabled(userRolesFeatureConfig,
                         RoleConstants.FEATURE_DICTIONARY.get("ROLE_DELETE"))
                     || !hasRequiredScopes(userRolesFeatureConfig,
-                        userRolesFeatureConfig?.scopes?.delete, allowedScopes),
+                        userRolesFeatureConfig?.scopes?.delete, allowedScopes);
```

Comment:
add organization type as well

Reply:
No need to add organization type as system roles are already handled as readonly by `systemRole` property.